rb/lib/selenium/webdriver/edge.rb

```diff
@@ -26,23 +26,11 @@ module Selenium
   module WebDriver
     module Edge
       def self.driver_path=(path)
-        warn <<-DEPRECATE.gsub(/\n +| {2,}/, ' ').freeze
```

Comment:
Now there's the best of both worlds, he just removed the deprecation warning and kept both ways of specifying the `driver_path`